include/Platform/Universal Windows/UWP/RTHelpers.h

```diff
@@ -893,22 +893,19 @@ static HString nsStrToHstr(NSString* str) {
         // We increment this just to make sure that we return an empty HString.
```

Comment:
I would need to modify StringHelpers.h to not include Starboard.h.  
I'll look into it, but, I'd prefer not to change more than I need to do for this bug fix.